### PR TITLE
Updating Service Fabric SDK to latest 6.1 release

### PR DESCRIPTION
**Pull request by samedder**
Changes include:
- All files related to Service Fabric 6.1 SDK
- Updates to History to reflect new version

### CODE REVIEW

**Comment by lmazuel** [comment]
> class ServiceFabricClientAPIsConfiguration(AzureConfiguration):

Why did you use the Azure generator? Should not be needed and will change your set of dependencies.

---
Refers to: azure-servicefabric/azure/servicefabric/service_fabric_client_ap_is.py:21 in 49fd2e3. [](commit_id = 49fd2e34123d1e0910bf798cbee7735ebcb0ac8d, deletion_comment = False)

File: azure-servicefabric/azure/servicefabric/service_fabric_client_ap_is.py [hunk 1]
```diff
@@ -10,20 +10,22 @@
...
+from msrest import Serializer, Deserializer
+from msrestazure import AzureConfiguration
 from .version import VERSION
 from msrest.pipeline import ClientRawResponse
+import uuid
 from . import models
 
 
-class ServiceFabricClientAPIsConfiguration(Configuration):
+class ServiceFabricClientAPIsConfiguration(AzureConfiguration):
```
Comment (lmazuel) [inline azure-servicefabric/azure/servicefabric/service_fabric_client_ap_is.py]: Why did you use the Azure generator? Should not be needed and will change your entire set of dependencies.

**Comment by lmazuel** [comment]
@samedder With latest Readme update, you're supposed to be able to just do:
```shell
autorest specification/servicefabric/data-plane/readme.md --python --python-mode=update --python-sdk-folders=C:/Git/azure-sdk-for-python
```
And this should be your exact client. If it's not, please tell me so I could fix the Readme.

**Comment by samedder** [comment]
@lmazuel I tried that command, ran into the following error:

> box-n-socks:source samedder$ autorest ./azure-rest-api-specs/specification/servicefabric/data-plane/Microsoft.ServiceFabric/stable/6.1/readme.md --python --python-mode=update --python-sdk-folders=./azure-sdk-for-python/
> AutoRest code generation utility [version: 2.0.4238; node: v9.3.0]
> (C) 2018 Microsoft Corporation.
> https://aka.ms/autorest
>    Loading AutoRest core      '/Users/samedder/.autorest/@microsoft.azure_autorest-core@2.0.4238/node_modules/@microsoft.azure/autorest-core/dist' (2.0.4238)
>    Loading AutoRest extension '@microsoft.azure/autorest.python' (~2.1.26)
>    Loading AutoRest extension '@microsoft.azure/classic-openapi-validator' (~1.0.9)
>    Loading AutoRest extension '@microsoft.azure/openapi-validator' (~1.0.2)
>    Loading AutoRest extension 'oav' (~0.4.20)
>    Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.38)
> [Exception] No input files provided.


Here's the [readme.md](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/servicefabric/data-plane/Microsoft.ServiceFabric/stable/6.1/readme.md) that I'm using.

**Comment by samedder** [comment]
Ah NVM, didn't realize I still needed to specify the file explicitly.

**Comment by samedder** [comment]
@lmazuel I tried updating the SDK using the mechanism you suggested and it appeared to produce the same output.

**Comment by lmazuel** [comment]
@samedder Shoud not. Let me check tomorrow if ARM is activated by default (should not!!!)

**Comment by samedder** [comment]
@lmazuel do you have an idea what we need to do to generate the SDK correctly?

**Comment by lmazuel** [comment]
@samedder I regenerated using the official command and latest RestAPI state:
autorest .\specification\servicefabric\data-plane\readme.md --python --pytho
n-sdks-folder=D:\VSProjects\azure-sdk-for-python-official --python-mode=update

This, as expected, doesn't use ARM, so I don't know what command you have used, but it's not the right one :/

I pushed the update into your branch, knowing that you can reset --hard if you don't like the content, but that's the official. I won't suggest to release something that is not in sync with RestAPI repo.
Thoughts?

**Comment by samedder** [comment]
@lmazuel I see the difference, I pointed directly to the 6.1 ReadME file, you pointed to the parent file in just the `data-plane` folder. Is it an issue that that file doesn't have references to the 6.1 tag?

**Comment by lmazuel** [comment]
The 6.1 readme should not exists actually :/. There should only one Readme per `resource-manager` or `data-plane` folder (that's the recommendation).

**Comment by lmazuel** [comment]
Otherwise, are you happy with the current content of this PR?

**Comment by samedder** [comment]
@lmazuel yes, sorry I've been OOF. Changes look similar to mine.

**Comment by samedder** [comment]
@lmazuel can these changes be merged today?

**Comment by lmazuel** [comment]
@samedder Just to be sure we're on the same page, you mean merge and release, or just merge?

**Comment by samedder** [comment]
Merge and release, as this will be the 6.1 release.

**Pull request merged**